### ISSUE TITLE
Generate dinamic SVG images from script

### ISSUE THREAD

**Issue by maxinteger** [zsebtanar/zsebtanar-proto#504]
Add new functions to create dynamic images 

possible way to add them to markdown is to convert the result into base64 then inject it as normal image

**Comment by vektor8891** [comment]
@maxinteger what do you think of this? https://tikzjax.com/

**Comment by vektor8891** [comment]
i found it here: https://github.com/mathjax/MathJax/issues/41

**Comment by maxinteger** [comment]
@vektor8891 Unfortunately tikzjax is just an experiment with zero support and code size is huge (double as the whole Zsebtanar app). But TiKZ looks promising as a vector graphics language so we can write an interpreter which translate basic operations into SVG or Canvas

**Comment by vektor8891** [comment]
@maxinteger i think it would consume too much energy to write such an interpreter. it would significantly slow down the process (there are *a lot* of features) and in fact, we would be reinventing the wheel... i just checked Brilliant, they generate SVG directly into HTML (example [here](https://brilliant.org/practice/calculating-area/?p=2)). In fact, they even let users control certain parameters and generate the SVG dynamically (example [here](https://brilliant.org/practice/calculating-area/?p=1)). can't we do something similar too?

**Comment by maxinteger** [comment]
@vektor8891 I agree implementing TiKZ would be a huge work. Another thing what we can do is to sanitize the SVG input with DOMPurify to prevent security problems.

**Comment by vektor8891** [comment]
@maxinteger i've started to write [some functions](https://gist.github.com/vektor8891/7acf7b658720270f553598917cb36359#svg-f%C3%BCggv%C3%A9nyek) that will be useful for SVG generation.
